sap-xmake-build.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash -l
+
+# This build script is only applicable to Spark without Hadoop and Hive
+
+PACKAGE_BRANCH="alti-branch-0.5"
+HADOOP_VERSION="2.7.4"
+SPARK_VERSION="2.1.1"
+SCALA_VERSION="2.11"
+LIVY_VERSION="0.5.1"
+
+curr_dir=`dirname $0`
+curr_dir=`cd $curr_dir; pwd`
+git_hash=""
+
+env | sort
+
+if [ "x${PACKAGE_BRANCH}" = "x" ] ; then
+  echo "error - PACKAGE_BRANCH is not defined. Please specify the branch explicitly. Exiting!"
+  exit -9
+fi
+
+echo "ok - extracting git commit label from user defined $PACKAGE_BRANCH"
+
+git_hash=$(git rev-parse HEAD | tr -d '\n')
+echo "ok - we are compiling livy branch $PACKAGE_BRANCH upto commit label $git_hash"
+
+echo "build - entire livy project in $curr_dir"
+
+if [ "x${HADOOP_VERSION}" = "x" ] ; then
+  echo "fatal - HADOOP_VERSION needs to be set, can't build anything, exiting"
+  exit -8
+fi
+
+echo "ok - building entire pkg with HADOOP_VERSION=$HADOOP_VERSION SPARK_VERSION=$SPARK_VERSION scala=$SCALA_VERSION"
+
+
+# PURGE LOCAL CACHE for clean build
+# mvn dependency:purge-local-repository
+
+########################
+# BUILD ENTIRE PACKAGE #
+########################
+# Default JDK version applied is 1.8 here.
+
+spark_profile_str=""
+if [[ $SPARK_VERSION == 1.* ]] ; then
+  spark_profile_str="-Pspark-1.6"
+elif [[ $SPARK_VERSION == 2.1.* ]] ; then
+  spark_profile_str="-Pspark-2.1"
+elif [[ $SPARK_VERSION == 2.2.* ]] ; then
+  spark_profile_str="-Pspark-2.2"
```

Comment:
I think the profile is missing for spark 2.3.2. https://github.com/Altiscale/incubator-livy/blob/sap-branch-0.5.1-alti/pom.xml#L1145 ? perhaps we need to review this later.